src/main/resources/view/LightTheme.css

```diff
@@ -117,7 +129,7 @@
 }
 
 .list-cell .label {
```

Comment:
Maybe we can add some padding so that the texts isn't too cluttered? E.g. `-fx-padding: 0px 0px 2px 0px;`? Same for the `#tags` css.